src/Object/Search/Result.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Friendica\Object\Search;
+
+use Friendica\Model\Search;
+
+/**
+ * A search result
+ *
+ * @see Search for defails
+ */
+class Result
```

Comment:
This is more specifically a Contact Search Result and I think the class name should reflect the specificity. Solr has generic search result entities with a title, a body and a URL that can refer to any entity type (page, user profile, book, etc...) but this isn't such a generic search result entity.